$OpenBSD: patch-pTk_Xlib_h,v 1.1 2011/05/30 19:08:35 matthieu Exp $

Note that you can't use #if !defined() here because this file is proccessed by a perl script that doesn't grok that.
--- pTk/Xlib.h.orig	Sat Feb 10 09:55:23 2007
+++ pTk/Xlib.h	Sun May 29 18:21:40 2011
@@ -1,6 +1,7 @@
 #ifndef _XLIB
 #define _XLIB
 #ifndef _XLIB_H_
+#ifndef _X11_XLIB_H_
 extern XFontStruct *XLoadQueryFont  _ANSI_ARGS_((Display *, const char *));
 extern XModifierKeymap *XGetModifierMapping  _ANSI_ARGS_((Display *));
 extern XImage *XCreateImage  _ANSI_ARGS_((Display *, Visual *, unsigned int, int, int, char *, unsigned int, unsigned int, int, int));
@@ -143,6 +144,7 @@ extern int XFreeFontNames _ANSI_ARGS_((char **));
 extern Window XGetSelectionOwner _ANSI_ARGS_((Display *, Atom));
 extern int XRectInRegion _ANSI_ARGS_((Region,int,int,unsigned,unsigned));
 extern int XSubtractRegion _ANSI_ARGS_((Region, Region, Region));
+#endif	/* _X11_XLIB_H_ */
 #endif /* _XLIB_H_ */
 extern int _XInitImageFuncPtrs _ANSI_ARGS_((XImage *image));
 #endif /* _XLIB */
